ApiReview/API.Polly.Extensions/NoDocs/Polly/PollyServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,15 @@
+// Assembly 'Polly.Extensions'
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Polly.Extensions.DependencyInjection;
+
+namespace Polly;
+
+public static class PollyServiceCollectionExtensions
+{
+    public static IServiceCollection AddResilienceStrategy<TKey, TResult>(this IServiceCollection services, TKey key, Action<ResilienceStrategyBuilder<TResult>> configure) where TKey : notnull;
```

Comment:
I think it would be nice to have some global strategy. Good example for this could be `HttpClient`, where calling `AddHttpClient()` adds a global `HttpClient` while `AddHttpClient(name)` adds named `HttpClient`. This would simplify the API for users that only need one strategy for one reason or another. The API itself could in theory be left mostly intact but just provide defaults for `AddResilienceStrategy` and `ResilienceStrategyProvider`.

A workaround for users that want only one strategy without the hassle of managing key is passing empty string, or 0 everywhere which I would say is not a very nice solution.